### DISCUSSION TITLE
the rawtype error about TableSaw in the demo Code

### DISCUSSION THREAD

**Discussion by xuhongxin** [deepjavalibrary/djl#2889] [Development]
![image](https://github.com/deepjavalibrary/djl/assets/1044109/80b3c60c-1f39-4560-99cd-28dc9c901c89)

`missing type arguments for generic class Column<T>
  where T is a type-variable:
    T extends Object declared in interface Column`

` where T is a type-variable:
    T extends Object declared in interface Column`

how to use tablesaw api?

**Comment by zachgk** [comment]
Some of the data types such as `Column` are parameterized. For example, if you have a List you need to know what is in the list such as `List<String>` or `List<Integer>`. Declaring a variable like `Column col = ...` is similar to doing `List list = ...`. Instead, you should either declare the type `Column<String> col = ...` or use a wildcard type like `Column<?> col = ...`.

For more information about using Tablesaw, we have some examples in the [D2L book](https://d2l.djl.ai/chapter_preliminaries/tablesaw.html?highlight=tablesaw). You can also look at tablesaw's own documentation
